### PR TITLE
Add linux arm64/ppc64le as well

### PR DESCRIPTION
**Pull request by lgray**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 build_platform:
   osx_arm64: osx_64
+  linux_aarch64: linux_64
```
Comment (xhochy) [inline conda-forge.yml:3]: Please also add `linux_ppc64le` while you're at it to keep both in sync.
```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Comment by lgray** [comment]
drat @xhochy could you provide some guidance here? Haven't mess with cuda + conda + non-intel archs before.

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Comment by lgray** [comment]
@xhochy just needed a rebase and a re-render and we're good. Looks like cuda for ppc64le and aarch64 comes later?

**Comment by lgray** [comment]
@aldanor @fhoehle @jakirkham @ksangeek could one of you please take a look and see what's up?

**Review by trivialfis — COMMENTED** [review]
Hi, thank you for working on this.  Can we omit the build for ppc for now?  We (xgboost) run basic tests on this platform using Travis but no developer has actual access to such machines.  Looking at the error, there seem to be issues somewhere in the pipeline causing incorrect inputs, specifically the label array is messed up, it might be in NumPy or in xgboost, we can't be sure.

**Comment by lgray** [comment]
OK - I'll drop it. Thanks!

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Review by hcho3 — APPROVED** [review]

**Comment by trivialfis** [comment]
I think I found the issue with tests on ppc64le: https://github.com/numpy/numpy/issues/21267 .

**Pull request merged**